staging/cert-manager-setup/templates/certificates.yaml

```diff
@@ -29,5 +32,16 @@ spec:
     - {{ . | quote -}}
   {{- end }}
 {{- end }}
+{{- if .ipAddresses }}
+  ipAddresses:
+  {{- range .ipAddresses }}
+    - {{ . | quote -}}
+  {{- end }}
+{{- end }}
+  keyAlgorithm: {{ .keyAlgorithm | default "rsa" }}
+  keyEncoding: {{ .keyAlgorithm | default "pkcs1" }}
+  keySize: {{ .keyAlgorithm | default "2048" }}
```

Comment:
Should you use `.keySize` and `.keyEnconfig` ?

Reply:
`. keyEncoding` is for encoding a private key
`. keySize` is the size of the private key.

but I see the copy paste mistake 🤦‍♂ 🙇 `.keyAlgorithm`